### PR TITLE
Resolve Symlinks

### PR DESCRIPTION
**Pull request by firesurfer**
See #310 

EDIT:
This PR solves only half of the problem. In order to fully solve the problem I think we should not do any assumptions on how the video device is actually called.

The reason for this edit is that we usually work with docker containers. We only mount our /dev/camerax devices into the container and not the /dev/videox. Additionally if mounted via `--device` a symlink in dev becomes a normal file entry in dev in the container. 

This PR at least solves the problem if /dev/videox exists and /dev/camerax (or whatever the symlink is called) exists in /dev.

### CODE REVIEW

**Comment by flynneva** [comment]
@firesurfer changes lgtm just I think CI failed because of a missing space between the // and comment.

Once you fix that I'll approve and merge this in

**Comment by firesurfer** [comment]
I just saw that the package does not provide a .clang-format file. Perhaps it might be useful to provide one. Because I have .clang-format in my workspace I use for our internal package which is then used by clang format. 
I try to fix the formatting manually for now.

**Comment by flynneva** [comment]
@firesurfer I just fixed the linter errors here, I'll wait for CI to pass then merge this in.

Regarding clang format: feel free to make a PR to enable it.

For now we use the ros2 defaults (cpplint and uncrustify). To fix your linter errors here I just ran the command `ament_uncrustify --reformat` from within the `usb_cam` directory and it fixed the linter errors.

**Review by flynneva — APPROVED** [review]

**Pull request merged**
